metadata.rb

```diff
@@ -4,7 +4,7 @@
 license          'Apache 2.0'
 description      'Installs/Configures datadog components'
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          '2.4.0'
+version          '2.4.1'
```

Comment:
Removed






On Fri, Jul 8, 2016 at 10:28 AM -0700, "Olivier Vielpeau" <notifications@github.com> wrote:












In metadata.rb:
> @@ -4,7 +4,7 @@
>  license          'Apache 2.0'
>  description      'Installs/Configures datadog components'
>  long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
> -version          '2.4.0'
> +version          '2.4.1'



Could you remove the version bump from the PR?



—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub, or mute the thread.